core/src/main/resources/org/apache/spark/ui/static/webui.css

```diff
@@ -194,6 +194,24 @@ span.additional-metric-title {
   color: #777;
 }
 
+#additional-metrics {
+  float: left;
+  padding-bottom: 10px;
+}
+
+#links {
+  float: left;
+  padding-left: 100px;
```

Comment:
This feels hacky, layout-wise. Can this be properly laid out in a table or something?